variables.tf

```diff
@@ -0,0 +1,31 @@
+variable "emr_release_label" {
+  description = "Version of AWS EMR to deploy with associated applicatoins"
+  default     = "emr-5.29.0"
```

Comment:
I think this should match the ingest-cluster.  Which is emr-5.24.1.

Reply:
changed